models/User.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+
+const UserSchema = new mongoose.Schema({
+    zprn: {
+        type: String,
```

Comment:
Need to change this to number and try what we were thinking earlier, about validating using string, but checking only for the digits in `Joi`

Reply:
I'll see to this